### PR TITLE
Remove old apps publishing code

### PR DESCRIPTION
**Pull request by islemaster**
I noticed today that this code no longer has an owner on our team and doesn't appear to be used anywhere.  This looks like a _very_ old build script. I believe this has been superseded by package.rake and its related code.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/script/bump-version.pl [hunk 1]
```diff
@@ -1,10 +0,0 @@
-#!/usr/bin/perl
-
-while (<>) {
-  if (/"version": *"(.*\.)(\d+)"/) {
-    $bumped = $2 + 1;
-    print "  \"version\": \"$1$bumped\",\n";
-  } else {
-    print;
-  }
-}
```
Comment (islemaster) [inline apps/script/bump-version.pl:10]: Who knew we had _Perl_ in our repo?

Comment (davidsbailey) [reply apps/script/bump-version.pl:10]: Perl?!? 🤪

**Review by davidsbailey — APPROVED** [review]
Thank you for the clean up!

**Comment by wjordan** [comment]
> This looks like a very old build script.

Source: https://github.com/code-dot-org/apps-archived/pull/304

**Pull request merged**